src/pages/home/subscription/subscription.styled.js

```diff
@@ -54,9 +54,9 @@ export const Link = styled.a`
 `;
 
 export const JoinButton = styled(Button)`
-  ${mixins.flexCenter()}
+  ${mixins.flexCenter()};
   ${(props) =>
-    props.isMobile
+    props.$isSmallScreen
```

Comment:
Use "transient" props to fix the following JS warning in the 
<img width="848" alt="Screen Shot 2021-08-24 at 3 05 51 PM" src="https://user-images.githubusercontent.com/123074/130550127-dedb9d58-22d4-4931-9892-f0c588129b78.png">
console.

Reply:
https://styled-components.com/docs/api#transient-props